application/tests/controllers/rss/Rss_test.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+class Rss_test extends TestCase
+{
+  /**
+   * @dataProvider provider
+   */
+  public function test_index($id, $expected)
+  {
+    $response = $this->request('GET', array('rss', 'index', $id));
+    foreach ($expected as $str) {
+      $this->assertStringContainsString($str, $response);
+    }
+
+    // Ugly: our test harness inserts messages like "got an instance" and "called the actual controller", before and after the actual RSS/XML response document.  Trim them off.
```

Comment:
Do we know where these come form? A quick grep of our own code base doesn't turn up anything, and searching on GitHub the contents of phpunit and ci-phpunit-test doesn't find anything here. I'd much rather we get rid of these at the source if at all possible than working around them with hacks.

(But thanks for the code comment, I would have been left wondering otherwise).